src/System.Security.Cryptography.Cng/ref/System.Security.Cryptography.Cng.cs

```diff
@@ -261,8 +260,8 @@ public sealed partial class DSACng : System.Security.Cryptography.DSA
         public System.Security.Cryptography.CngKey Key { get { throw null; } }
         public override string KeyExchangeAlgorithm { get { throw null; } }
         public override System.Security.Cryptography.KeySizes[] LegalKeySizes { get { throw null; } }
-        public override byte[] CreateSignature(byte[] rgbHash) { throw null; }
         public override string SignatureAlgorithm { get { throw null; } }
+        public override byte[] CreateSignature(byte[] rgbHash) { throw null; }
         protected override void Dispose(bool disposing) { }
         public override System.Security.Cryptography.DSAParameters ExportParameters(bool includePrivateParameters) { throw null; }
```

Comment:
Should we be adding overrides of the base type? For example:
```C#
        public override byte[] ExportEncryptedPkcs8PrivateKey(System.ReadOnlySpan<byte> passwordBytes, System.Security.Cryptography.PbeParameters pbeParameters) { throw null; }
        public override byte[] ExportEncryptedPkcs8PrivateKey(System.ReadOnlySpan<char> password, System.Security.Cryptography.PbeParameters pbeParameters) { throw null; }
```

Reply:
I'm content with whatever the tool wants to do.  Declaring overrides on sealed types doesn't have any real impact (unless the base member was declared abstract, then they're required).